br/pkg/lightning/backend/external/sort_test.go

```diff
@@ -148,7 +148,7 @@ func TestGlobalSortLocalWithMerge(t *testing.T) {
 		startKey = BytesMin(startKey, s.Min.Clone())
 		endKey = BytesMax(endKey, s.Max.Clone().Next())
 	}
-
+	mergeMemSize := (rand.Intn(10) + 1) * 100
```

Comment:
what this “mergeMemSize := (rand.Intn(10) + 1) * 100” means，why here use a random func?

Reply:
Make the test run more cases to check different memSize/blockSize of writer can be used for writer.
When memSize is small enough, we will reach line 109 in onefile writer, hopefully the bug will reproduce.